core/arch/arm/plat-ti/conf.mk

```diff
@@ -15,6 +15,7 @@ $(call force,CFG_PL310_LOCKED,y)
 $(call force,CFG_SECURE_TIME_SOURCE_REE,y)
 arm32-platform-cpuarch		:= cortex-a9
 else
+CFG_OTP_SUPPORT ?= y
```

Comment:
Right, when AM43xx gets support for this then we can safely drop CFG_OTP_SUPPORT.